plugin/src/main/kotlin/org/gradle/github/dependency/extractor/internal/DependencyExtractorService.kt

```diff
@@ -110,14 +110,17 @@ abstract class DependencyExtractorService :
         val repositoryLookup = RepositoryUrlLookup(details, result)
         val rootComponent = result.rootComponent
         val projectIdentityPath = (rootComponent.id as? DefaultProjectComponentIdentifier)?.identityPath?.path
-        @Suppress("FoldInitializerAndIfToElvis") // Purely for documentation purposes
-        if (projectIdentityPath == null) {
-            // TODO: We can actually handle this case, but it's more complicated than I have time to deal with
-            // TODO: See the Gradle Enterprise Build Scan Plugin: `ConfigurationResolutionCapturer_5_0`
+
+        if (projectIdentityPath == null && rootComponent.dependencies.isEmpty()) {
+            // Not a project configuration, and no dependencies to extract: can safely ignore
```

Comment:
Why is a `ResolveConfigurationDependenciesBuildOperationType` event issued in this case then?

Reply:
This happens when a configuration is resolved that doesn't have any dependencies configured. I'm pretty sure the main case this happens is when resolving a `classpath` configuration when no plugins (or other classpath dependencies) have been declared.